### PR TITLE
Fix 3D Touch swiping issue

### PR DESCRIPTION
**Pull request by webraptor**

### CODE REVIEW

**Comment by AlexVegner** [comment]
@webraptor Hi @webraptor, 
I will verify your solution on my iPhone 7, 6+ later today.

**Comment by webraptor** [comment]
Allright, already tested on simulator and 2 different devices with and without 3D touch

**Comment by alexbrillant** [comment]
Thanks alot ! @webraptor  @AlexVegner What issue is it fixing ?

**Comment by AlexVegner** [comment]
Hi @webraptor @alexbrillant, 
This solition doen't work for me. 
On iPhone 7 issue https://github.com/alexbrillant/react-native-deck-swiper/issues/33 is reproducible. Swiper interceps Buttons tap on iPhone 7 with 3d touch. 
Only`onM oveShouldSetPanResponder: (event, gestureState) => false` help with this issue.

**Comment by webraptor** [comment]
And that one causes swiping not to work at all on simulator and all devices. Does swiping work for you in that scenario ?

**Comment by AlexVegner** [comment]
@webraptor 
> And that one causes swiping not to work at all on simulator and all devices. Does swiping work for you in that scenario ?
Yes, with https://github.com/alexbrillant/react-native-deck-swiper/pull/40/ swipe worked for my case. Tested on 6+, 7 devices. 
I will create test project later today, so we will discuss this issue based on same code base.

**Comment by webraptor** [comment]
I was able to fix swiping and now working to fix force/3d touch on inner buttons. Will create a PR when finished.

File: Swiper.js [hunk 1]
```diff
@@ -110,6 +110,7 @@ class Swiper extends React.Component {
     this._panResponder = PanResponder.create({
       onStartShouldSetPanResponder: (event, gestureState) => true,
       onMoveShouldSetPanResponder: (event, gestureState) => false,
```
Comment (webraptor) [inline Swiper.js]: This line partly fixes 3D touch issues.

File: Swiper.js [hunk 2]
```diff
@@ -110,6 +110,7 @@ class Swiper extends React.Component {
...
+      onMoveShouldSetPanResponderCapture: (evt, gestureState) => gestureState.dx != 0 && gestureState.dy != 0,
```
Comment (webraptor) [inline Swiper.js]: This line fixed manual swiping around (pan responder) on 3D touch devices.

**Comment by webraptor** [comment]
Added a few extra comments on the lines affected by this PR, with info on what they actually do. 
Changes actually fix swiping and 3D touching, but following info should be taken into consideration.

**Inline touch not working?**
If within the swipe cards (aka panresponder) one has touchable items that do not work, here are some solutions:
1. Add **hitslop** on touchable items (touchableOpacity, touchableWithoutFeedback, whatever you have etc) that are directly within the render card method.
2. If the touchables mentioned at point (1.) are within an Animated.View class, **use _onPressOut_ in favor of _onPress_** in which isn't triggered by 3D touch devices.

I am sure there are still some other scenarios, but at least we now know how to fix basic behaviors.

**Comment by AlexVegner** [comment]
@webraptor @alexbrillant 
onMoveShouldSetPanResponder: (event, gestureState) => false,
Fix the issue, all types of buttons and swipe works with this PR on iPhone 7, etc.
https://github.com/AlexVegner/test-react-native-deck-swiper/blob/master/App.js#L19
Thanks.

File: Swiper.js (hunk 1 above)
Comment (AlexVegner) [inline Swiper.js]: Thanks!

**Comment by AlexVegner** [comment]
@alexbrillant You could close #33

**Pull request merged**